src/item_factory.cpp

```diff
@@ -1048,15 +1067,43 @@ void Item_factory::load( islot_comestible &slot, JsonObject &jo )
     } else {
         jo.read( "nutrition", slot.nutr );
     }
+
+    // any specification of vitamins suppresses use of material defaults @see Item_factory::finalize
+    if( jo.has_int( "vitamins" ) ) {
+        // convenience syntax for setting all vitamins concurrently
+        for( auto &v : vitamin::all() ) {
```

Comment:
All vitamins in equal amounts is rather silly, but there's no reason that a carefully made dish with a variety of ingredients from various food groups wouldn't be able to meet all the character's various vitamin needs to varying degrees.